web/src/components/pages/datasets/dataset-download-email-prompt/test-utils.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  ByRoleMatcher,
+  ByRoleOptions,
+  Matcher,
+  SelectorMatcherOptions,
+  fireEvent,
+} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+type downloadDatasetParams = {
```

Comment:
Types should start with a capital letter
```suggestion
type DownloadDatasetParams = {
```